pipelines/LSSTCam/DRP-FL.yaml

```diff
@@ -111,13 +111,19 @@ tasks:
   skyCorr:
     class: lsst.pipe.tasks.skyCorrection.SkyCorrectionTask
     config:
-      doSky: false
-      doBgModel2: false
       maskObjects.detection.isotropicGrow: false
       maskObjects.detection.thresholdType: stdev
       bgModel1.xSize: 81.92  # 8192 pixels
       bgModel1.ySize: 81.92
       bgModel1.pixelSize: 0.01
+      bgModel1.doSmooth: false
+      doSky: true
+      doBgModel2: true
+      bgModel2.xSize: 40.96  # 4096 pixels
+      bgModel2.ySize: 40.96
+      bgModel2.pixelSize: 0.01  # mm per pixel
```

Comment:
Could add the `# mm per pixel` comment to `bgModel1.pixelSize: 0.01` as well (but I'm not insisting here!)

Reply:
Added!